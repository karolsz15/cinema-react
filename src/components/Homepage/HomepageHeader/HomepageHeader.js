import React from 'react';
import {Link} from 'react-router-dom';

const homepageHeader = props => (

    <React.Fragment>
            <div className="header" style={{background: `url(${props.bigImage})` }}>
                <div className="top-header">
                    <div className="logo">
                        <a href="/"><img src="images/logo4.png" alt="" /></a>
                        <p>Ticket booking app</p>
                    </div>
                    <div className="clearfix"></div>
                </div>
                <div className="header-info">
                    <h1>{props.title}</h1>
                    <p className="age">{props.cast}</p>
                    <p className="review reviewgo">Genre: {props.genres} </p>
                    <p className="review">Release: {props.release} </p>
                    <p className="special"> {props.summary} </p>
                    <div className="headerButtons">
                        <Link onClick={props.trailerClicked} className="video"><i className="video1"></i>TRAILER</Link>
                        <Link to="/booking" className="book"><i className="book1"></i>TICKET</Link>
                    </div>

                </div>
            </div>
    </React.Fragment>
);

export default homepageHeader;