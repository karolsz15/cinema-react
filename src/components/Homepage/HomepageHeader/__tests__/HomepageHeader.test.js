import React from 'react';
import { shallow } from 'enzyme';
import HomepageHeader from '../HomepageHeader';

let wrapper = shallow(<HomepageHeader />);

describe('HomepageHeader component', () => {
  it('should render the HomepageHeader correctly', () => {   
    expect(wrapper).toMatchSnapshot();
  });
});