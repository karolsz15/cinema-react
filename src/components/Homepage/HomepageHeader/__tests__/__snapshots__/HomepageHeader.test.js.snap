// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageHeader component should render the HomepageHeader correctly 1`] = `
<div
  className="header"
  style={
    Object {
      "background": "no-repeat top url(undefined)",
    }
  }
>
  <div
    className="top-header"
  >
    <div
      className="logo"
    >
      <a
        href="/"
      >
        <img
          alt=""
          src="images/logo4.png"
        />
      </a>
      <p>
        Ticket booking app
      </p>
    </div>
    <div
      className="clearfix"
    />
  </div>
  <div
    className="header-info"
  >
    <h1 />
    <p
      className="age"
    />
    <p
      className="review reviewgo"
    >
      Genre: 
       
    </p>
    <p
      className="review"
    >
      Release: 
       
    </p>
    <p
      className="special"
    >
       
       
    </p>
    <div
      className="headerButtons"
    >
      <div
        className="videoButton"
      >
        <i
          className="video1"
        />
        TRAILER
      </div>
      <Link
        className="bookButton"
        to="/booking"
      >
        <i
          className="book1"
        />
        TICKET
      </Link>
    </div>
  </div>
</div>
`;
